### ISSUE TITLE
entity_path_prefix not working for log_file()

### ISSUE THREAD

**Issue by EtaLoop** [rerun-io/rerun#8344]
**Describe the bug**
`entity_path_prefix` does not seem to work on `rerun.log_file_from_path` on 0.20.3.

**To Reproduce**
```
rec = rr.new_recording("app", spawn=True)

rr.log_file_from_path(
    "cube.stl",
    entity_path_prefix="test",
    recording=rec.to_native(),
)

```

**Comment by teh-cmc** [comment]
I don't believe we have a builtin STL loader, so I assume this is a custom one you made.

All `entity_path_prefix` does is pass a flag of the same name to your dataloader -- from there it's your job to do whatever you'd like with that information.

See https://rerun.io/docs/reference/data-loaders/overview:
![Image](https://github.com/user-attachments/assets/38b94a28-5b44-4d3c-92ca-1424b2bc093e)

(Closing, feel free to re-open if needed)

**Comment by EtaLoop** [comment]
Here : https://rerun.io/docs/getting-started/data-in/open-any-file it says that stl is supported.
So I am wrong if I'm saying that there is a DataLoader for stl ?
